### ISSUE TITLE
runAppEngine throws ProcessException on windows

### ISSUE THREAD

**Issue by tasosx** [dart-lang/appengine#37]
Fails to find "/bin/hostname" (no bin folder for windows..., runs fine on Linux)

To reproduce it, follow Søren Gjesse comment on how to run an app locally: https://groups.google.com/a/dartlang.org/forum/#!msg/cloud/zrxgOHFz_lA/q5CdLLQPBAgJ

```
Unhandled exception:
Uncaught Error: ProcessException: The system cannot find the file specified.

  Command: /bin/hostname
Stack Trace:
#0      _ProcessImpl._runAndWait (dart:io-patch/process_patch.dart:474)
#1      _runNonInteractiveProcessSync (dart:io-patch/process_patch.dart:626)
#2      Process.runSync (dart:io-patch/process_patch.dart:64)
#3      _getHostname (package:appengine/src/server/server.dart:20:14)
#4      AppEngineHttpServer.AppEngineHttpServer (package:appengine/src/server/server.dart:36:26)
#5      runAppEngine.<anonymous closure> (package:appengine/src/appengine_internal.dart:134:31)
#6      withAppEngineServicesInternal.<anonymous closure>.<anonymous closure> (package:appengine/src/appengine_internal.
dart:123:22)
#7      _rootRunUnary (dart:async/zone.dart:902)
#8      _CustomZone.runUnary (dart:async/zone.dart:804)
#9      _Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:494)
#10     _Future._propagateToListeners (dart:async/future_impl.dart:577)
#11     _Future._completeWithValue (dart:async/future_impl.dart:368)
#12     _Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:422)
#13     _rootRun (dart:async/zone.dart:895)
#14     _CustomZone.run (dart:async/zone.dart:796)
#15     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:731)
#16     _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#17     _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#18     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#19     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)

#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:886)
#1      _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#2      _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)
```

**Comment by mkustermann** [comment]
Thank you for your report. As Soeren mentions on the mailing list "This is all experimental". If the code runs inside a docker container this issue should never occur. Running code outside a docker container is not supported at the moment, but might come at some point.

**Comment by walberleal** [comment]
tenho o mesmo problema. praticamente impossível debugar algo na parte servidora :(

**Comment by sgjesse** [comment]
Fixed in 37afc826814b6dd8e53d33f755676dc717703f89
